whylog/log_reader/searchers.py

```diff
@@ -42,6 +42,12 @@ def _deduce_offset(self, investigation_step):
             if investigation_step.is_line_in_search_range(line_content):
                 return line_offset + len(line_content) + 1
 
+    def _find_offset_by_date(self, opened_file, date):
```

Comment:
You can't assume that you will finding by date, it may be int, string etc. Rename this method to more generic name.